src/ar.hrl

```diff
@@ -141,8 +141,8 @@
 %% @doc Never have more than this number of peers (new peers excluded).
 -define(MAXIMUM_PEERS, 20).
 
-%% Maximum number of requests allowed by an IP in any 30 second period.
--define(MAX_REQUESTS, 450).
+%% Maximum allowed number of accepted requests per minute per IP.
+-define(DEFAULT_ACCEPT_REQUESTS_PER_MINUTE_LIMIT, 2000).
```

Comment:
I'm not comfortable with more than doubling this limit. Where did 2000 come from? Already 450 sounds really high to me.

Reply:
The important bit here is to make it configurable so that we can use it in our gateway so we agreed to not change the default for now, will fix it up.